### DISCUSSION TITLE
[Request] PlaceholderAPI relational placeholders support

### DISCUSSION THREAD

**Discussion by Drc-DEV** [Zrips/CMI#1417] [Feature Requests]
Yes there is another issue about this which was closed in november, I know, but the problem still persists.

In order to make relational placeholders work you have to add compatibility to them, because "PlaceholderAPI does not automatically parse relational placeholders on its own when normal placeholders are parsed!"

More info about how to implement those in this update:
https://www.spigotmc.org/resources/placeholderapi.6245/update?update=167267

**Comment by Zrips** [comment]
Well... For relational placeholders to work, we need 2 player objects. Currently system is set to handle only one and all of the commands passing one or no player objects at all when translating placeholders. This is quite specific thing to have.
How would we determine which 2 players we should compare too?

**Comment by Drc-DEV** [comment]
For example, in the Tablist configuration the two players would be the player displayed in the tablist and the player who is viewing the tablist.

In chat the two players would be the message sender and the recipient.

DeluxeChat already implemented the relational placeholders about one year ago or so.

**Comment by Drc-DEV** [comment]
Any news on this? It's really necessary for things like Factions' relational colors to work. @Zrips
